### PR TITLE
New Currency Cheat Sheet

### PR DESCRIPTION
**Pull request by Midhun-Jo-Antony**
**What does your Instant Answer do?**
A Cheat Sheet for currencies of different countries

**What is the data source for your Instant Answer? (Provide a link if possible)**
Wikipedia,
sourceUrl: https://en.wikipedia.org/wiki/List_of_countries_and_capitals_with_currency_and_language

**Why did you choose this data source?**
Reliable and easy-to-understand

**Are there any other alternative (better) data sources?**
No

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)
Anyone

**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**
Yes, https://duck.co/ia/view/6441

**Are you having any problems? Do you need our help with anything?**
Not yet

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)
c1
c2

-----
IA Page: https://duck.co/ia/view/currency_cheat_sheet

### CODE REVIEW

**Comment by gautamkrishnar** [comment]
#1611 duckdukhackcek

**Comment by moollaza** [comment]
@Midhun-Jo-Antony Hey!

Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

File: share/goodie/cheat_sheets/json/currency.json [hunk 1]
```diff
@@ -0,0 +1,235 @@
+{
+    "id": "currency_cheat_sheet",
+    "name": "Currencies",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/currency.json]: Let's make this a little more descriptive: "World Currencies"

File: share/goodie/cheat_sheets/json/currency.json [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    "description": "Currency Cheatsheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/currency.json]: Let's make this also a little more descriptive: "A list of countries across the world and their associated currency"

**Comment by Midhun-Jo-Antony** [comment]
@moollaza Changes Updated

**Comment by moollaza** [comment]
@Midhun-Jo-Antony sorry for the delay! Thanks for updating this. I hadn't realized before that you were using a Wikipedia table as the source. We're trying to avoid re-creating those tables as Cheat Sheets, because we have other plans to expose that information. I'll let this one by for now, but please be aware that we may remove this later when we do a better job of surfacing Wiki tables.

How does that sound?

**Comment by moollaza** [comment]
Also, it seems like there's some inconsistent casing on `Dollar` vs `dollar` -- I think they should all be lowercased. I can fix that pretty easily. Otherwise, this LGTM :+1:

**Comment by gautamkrishnar** [comment]
:+1: Congrats! @Midhun-Jo-Antony  :smile:

**Comment by moollaza** [comment]
@Midhun-Jo-Antony congrats! It's live: https://duckduckgo.com/?q=currency+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by Midhun-Jo-Antony** [comment]
@moollaza Thank you very much for making that PR live.

**Pull request merged**